MidiTools/src/main/java/com/mobileer/miditools/synth/SineOscillator.java

```diff
@@ -48,7 +48,7 @@ public static float fastSin(float currentPhase) {
     public float render() {
         // Convert raw sawtooth to sine.
         float phase = incrementWrapPhase();
-        return fastSin(phase) * getAmplitude();
+        return fastSin(phase) * 2* getAmplitude();
```

Comment:
Add a comment here like

// Manually multiply by 2 to get the volume of the sine wave similar to the saw tooth